src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java/CheckstyleJavaParser.g4

```diff
@@ -0,0 +1,858 @@
+/*
+ [The "BSD licence"]
+ Copyright (c) 2013 Terence Parr, Sam Harwell
+ Copyright (c) 2017 Ivan Kochurkin (upgrade to Java 8)
+ All rights reserved.
```

Comment:
Can you use the "suggested change" feature to show me the correct way to do this, while also maintaining the original copyright information? I have never added myself as the author to anything. Should I just add `Author: Nick Mancuso (upgrade to Java 16)` ?

Reply:
@romani Shouldn't all files have the Checkstyle copyright at the very least?
I've never done anything with copyrights, so I don't know what is right here.

Reply:
All files are same copyright.

Reply:
So, completely replace existing header with Checkstyle header?

Reply:
I have replaced the ANTLR header with the Checkstyle header. I think this is appropriate in this case, since we have made enough additions and changes that this work is loosely based on the base grammar from ANTLR, at best.